Automattic/wp-calypso
# Use latest version of notifications-panel

**Test Instructions**
* Open notifications in desktop mode.
* Click on a notification to open the detail view.
* Change filters at the top of the list view.
* The detail view should close.
